forem/forem
# noindex google any comment permalinks with negative score

## What type of PR is this? (check all applicable)


## Description

Currently comment permalinks are still indexed even if a mod/admin has deemed it low quality or spam. This PR adds the functionality to add the proper meta tag in this situation.

## QA Instructions, Screenshots, Recordings

Sanity check this logic. If inspecting manually, turn off service workers to see what the bots see.

### UI accessibility concerns?

This should not affect a11y.

## Added tests?

      have not been included_

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/) - This should include an admin guide update.
